### ISSUE TITLE
Restaurant code review

### ISSUE THREAD

**Issue by mandyWW** [james-d12/whitehatbootcamp#3]
Great job James, well done. Comments & questions below:

**Accessibility**
- clear font
- `+` button on Home Page does not have `aria-label` to explain what  it is doing. Also should be nearer the start of the page to save screen reader having to read the whole page before finding it

**General**
- use of HTTPS & certificates, nice!
- good structure of code into separate directories
- great to see use of fetch() API & understanding of async code
- great to see evidence of Cypress system tests
- I notice you are using PUT for edit, I like it, very RESTful. However, as you are using PUT you could remove the /edit from the url as PUT implies an update. /edit only needed if your APIs just use GET and POST
- need to include client side validation for form data (e.g. check valid url)
- need to include server side validation

**Questions:**
How are you using cookies/sessions?
